docker/test/stateful/run.sh

```diff
@@ -87,7 +87,7 @@ function start()
             tail -n1000 /var/log/clickhouse-server/clickhouse-server.log
             break
         fi
-        timeout 120 service clickhouse-server start
+        timeout 120 sudo -E -u clickhouse /usr/bin/clickhouse-server --config /etc/clickhouse-server/config.xml --daemon --pid-file /var/run/clickhouse-server/clickhouse-server.pid
```

Comment:
Write `sudo clickhouse start`, it is much better than `service`.

Reply:
And it doesn't preserve `env` variables as well